android/src/main/java/ly/count/android/sdk/react/CountlyReactNative.java

```diff
@@ -224,22 +225,28 @@ private void populateConfig(JSONObject _config) {
             }
             if (_config.has("pushNotification")) {
                 JSONObject pushObject = _config.getJSONObject("pushNotification");
-                int messagingMode = Integer.parseInt(pushObject.getString("tokenType"));
-                channelName = pushObject.getString("channelName");
-                channelDescription = pushObject.getString("channelDescription");
-                if (pushObject.has("accentColor")) {
-                    setHexNotificationAccentColor(pushObject.getString("accentColor"));
-                }
-
-                if (messagingMode == 0) {
-                    CountlyReactNative.messagingMode = Countly.CountlyMessagingMode.PRODUCTION;
-                } else {
+                if (!pushObject.toString().equals("{}")) {
+                    int messagingMode = Integer.parseInt(pushObject.getString("tokenType"));
```

Comment:
what happens if these things not exist, does not parseInt going to throw something? Or if they definitely exist and guaranteed than no problem